pipeline-model-api/src/main/java/org/jenkinsci/plugins/pipeline/modeldefinition/ast/ModelASTStages.java

```diff
@@ -14,7 +14,7 @@
  *
  * @author Andrew Bayer
  */
-public final class ModelASTStages extends ModelASTElement {
+public final class ModelASTStages extends ModelASTElement{
```

Comment:
formatting nitpick. =) If you could revert it back to `extends ModelASTElement {`, that'd be great.